packages/squiggle-lang/src/ast/utils.ts

```diff
@@ -8,8 +8,20 @@ export function locationContains(location: LocationRange, offset: number) {
 
 export function isBindingStatement(
   statement: ASTNode
-): statement is Extract<ASTNode, { type: "LetStatement" | "DefunStatement" }> {
+): statement is Extract<
+  ASTNode,
+  { type: "LetStatement" | "DefunStatement" | "DecoratedStatement" }
+> {
   return (
-    statement.type === "LetStatement" || statement.type === "DefunStatement"
+    statement.type === "LetStatement" ||
+    statement.type === "DefunStatement" ||
+    statement.type === "DecoratedStatement"
   );
 }
+
+export function undecorated(node: ASTNode) {
```

Comment:
Called "undecorated" and not "undecoratedStatement" because we'll get decorators for `KeyValue` pairs in dicts and other nodes soon.

Reply:
Maybe ``removeDecorations``?